javaparser-testing/src/test/resources/com/github/javaparser/bdd/visitor_scenarios.story

```diff
@@ -73,4 +73,4 @@ Given a CompilationUnit
 Given a VoidVisitorAdapter with a visit method that asserts sensible line positions
 When file "JavaConcepts.java" is parsed
 When the CompilationUnit is visited by the PositionTestVisitor
-Then the total number of nodes visited is 1427
```

Comment:
Although it is freaking annoying, it did make me notice today that the node counter was broken.

Wait, that's not important :) So yeah, I guess we can remove it.